### ISSUE TITLE
loaded plugin no accessories?

### ISSUE THREAD

**Issue by scheveningen** [nfarina/homebridge-sonos#33]
I try to install the plugin, But it won't work. I'm sure I have a small error in my config.json file.
But I can not figure it out. Install the plugin 2 times to be sure. Room name is the same as the name in my sonos app. 

[Sat Jul 02 2016 16:49:32 GMT+0000 (UTC)] Loaded plugin: homebridge-sonos
[Sat Jul 02 2016 16:49:32 GMT+0000 (UTC)] Registering accessory 'homebridge-sonos.Sonos'
[Sat Jul 02 2016 16:49:32 GMT+0000 (UTC)] ---
[Sat Jul 02 2016 16:49:32 GMT+0000 (UTC)] Loaded config.json with 0 accessories and 1 platforms.
[Sat Jul 02 2016 16:49:32 GMT+0000 (UTC)] ---

config.json file

{
    "description": "Configuration file for Domoticz platform.",
    "platforms": [
        {
            "platform": "Domoticz",
            "name": "Domoticz",
            "server": "127.0.0.1",
            "port": "8080"
        }
{
  "accessories": [
    {
      "accessory": "Sonos",
      "name": "Bedroom Speakers",
      "room": "Woonkamer" 
    }
  ]
}

**Comment by gizbod** [comment]
I think you are missing ], between platforms and accessories.

**Comment by gizbod** [comment]
Like this

{
"description": "Configuration file for Domoticz platform.",
"platforms": [
{
"platform": "Domoticz",
"name": "Domoticz",
"server": "127.0.0.1",
"port": "8080"
}],
"accessories": [
{
"accessory": "Sonos",
"name": "Bedroom Speakers",
"room": "Woonkamer" 
}
]

**Comment by scheveningen** [comment]
Still the same, I copy past the text using nano config.json Save the file, reboot domoticz and start homebridge and I get the same problem. "Loaded config.json with 0 accessories and 1 platforms."


[Sun Jul 03 2016 20:18:49 GMT+0000 (UTC)] Registering platform 'homebridge-edomoticz.eDomoticz'
[Sun Jul 03 2016 20:18:50 GMT+0000 (UTC)] ---
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] Loaded plugin: homebridge-sonos
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] Registering accessory 'homebridge-sonos.Sonos'
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] ---
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] Loaded config.json with 0 accessories and 1 platforms.
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] ---
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] Loading 1 platforms...
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] [Domoticz] Initializing eDomoticz platform...
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] [Domoticz] Fetching Domoticz lights and switches...
[Sun Jul 03 2016 20:18:53 GMT+0000 (UTC)] Loading 0 accessories...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 031-45-154 │     
    └────────────┘

**Comment by gizbod** [comment]
Post your full config blurring out any passwords

**Comment by scheveningen** [comment]
I just started with domoticz and homebridge. The installation of domoticz and homebridge went well. I use the app insteon+ after I started homebridge I see the homebridge in the app. So that is working fine. 
The first thing I want to get working is the songs plugin. 

Below the complete config.json

{
"platform": "Domoticz",
"name": "Domoticz",
"server": "127.0.0.1",
"port": "8080"
}],
"accessories": [
{
"accessory": "Sonos",
"name": "Bedroom Speakers",
"room": "Woonkamer"
}
]

**Comment by scheveningen** [comment]
still no luck, still no accessory?

**Comment by jochem725** [comment]
@scheveningen The config file you posted is not valid, I think this should solve your problem:

```
{
	"bridge": {
		"name": "Homebridge",
		"username": "CC:22:3D:E3:CE:30",
		"port": 51826,
		"pin": "031-45-154"
	},
	"description": "Configuration file for Domoticz platform.",
	"platforms": [{
		"platform": "Domoticz",
		"name": "Domoticz",
		"server": "127.0.0.1",
		"port": "8080"
	}],
	"accessories": [{
		"accessory": "Sonos",
		"name": "Bedroom Speakers",
		"room": "Woonkamer"
	}]
}
```

**Comment by scheveningen** [comment]
still no luck. BUT below is the way is start homebridge,

Tuesday, 12 July 2016,  8:25:17 pm UTC
Linux 4.1.18+ armv6l GNU/Linux
Uptime.............: 0 days, 00h20m27s
Memory.............: 286700kB (Free) / 445380kB (Total)
Load Averages......: 0.07, 0.05, 0.06 (1, 5, 15 min)
Running Processes..: 62
IP Addresses.......: 192.168.178.17

pi@raspberrypi:~$ homebridge

And if I edit the config.json I type,
pi@raspberrypi:~$ nano config.json


Is there a change that I edit the config.json in the wrong directory? It is just a guess.

**Comment by jochem725** [comment]
@scheveningen I think the correct directory for the `config.json` file should be `/home/pi/.homebridge` in your case. To edit the correct config file you should use `nano /home/pi/.homebridge/config.json`

**Comment by scheveningen** [comment]
YES!!! this is the problem, I edit the wrong config.json! But now there is a small new problem.
[ Error writing /home/pi/.homebridge/config.json: Permission denied ]

**Comment by jochem725** [comment]
@scheveningen You can write to that file using `sudo nano /home/pi/.homebridge/config.json`

**Comment by scheveningen** [comment]
@jochem725 sorry for the late replay, but is work fine now! Thank you so much.
